### ISSUE TITLE
Error parsing '7.10.3b' as a Version

### ISSUE THREAD

**Issue by HuStmpHrrr** [haskell/haskell-platform#235]
for Debian 7.10.3 64bit intel cpu version of ghc, hptool gives this error. 

it sounds to be the `b` is the problem, isn't it? the download link is [here](https://www.haskell.org/ghc/download_ghc_7_10_3).

**Comment by gbaz** [comment]
Yep, hptool tries to parse the string into a proper version, and the 'b' won't work. You could rename it to 7.10.3.1 I suppose, or just delete the 'b' I suppose, depending on what you want to do...

**Comment by HuStmpHrrr** [comment]
yeah, that's what i did. it's not very nice since the `b` version is the exact name comes from the download site. i will try to investigate and see if it can be by passed.
